imports/plugins/core/dashboard/client/components/OperatorLanding.js

```diff
@@ -14,7 +14,7 @@ import ShopLogoWithData from "/imports/client/ui/components/ShopLogoWithData/Sho
 function OperatorLanding() {
   return (
     <Fragment>
-      <Helmet title="Operator UI" />
+      <Helmet title="Reaction" />
```

Comment:
What's the reasoning behind hard-coding in "Reaction" here

Reply:
Hard coded value has been removed and replaced with a default value.